src/main/ruby/truffleruby/core/gc.rb

```diff
@@ -76,31 +76,32 @@ def garbage_collect
   end
 
   def self.stat(option = nil)
-    time, count, minor_count, major_count, unknown_count, heap, memory_pool_names, memory_pool_info = Primitive.gc_stat()
+    time, count, minor_count, major_count, unknown_count, heap, memory_pool_names, memory_pool_info = Primitive.gc_stat
     used, committed, init, max = heap
 
     # Initialize stat for statistics that come from memory pools, and populate it with some final stats
     stat = {
+      committed: committed,
       count: count,
-      time: time,
-      minor_gc_count: minor_count,
-      major_gc_count: major_count,
-      unknown_count: unknown_count, # if nonzero, major or minor count needs to be updated for this GC case
       heap_available_slots: committed,
-      heap_live_slots: used,
       heap_free_slots: committed - used,
-      used: used,
-      committed: committed,
+      heap_live_slots: used,
       init: init,
+      last_committed: 0,
+      last_init: 0,
+      last_max: 0,
+      last_used: 0,
+      major_gc_count: major_count,
       max: max,
-      peak_used: 0,
+      minor_gc_count: minor_count,
       peak_committed: 0,
       peak_init: 0,
       peak_max: 0,
-      last_used: 0,
-      last_committed: 0,
-      last_init: 0,
-      last_max: 0,
+      peak_used: 0,
+      time: time,
+      total_allocated_objects: 0,
+      unknown_count: unknown_count, # if nonzero, major or minor count needs to be updated for this GC case
+      used: used,
```

Comment:
Could you undo the reordering? The order is intentionally close to what MRI has and also makes it easy by having `*used` and `*committed` next to each other.